app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/DatabaseTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+import com.github.HumanLearning2021.HumanLearningApp.datamodel.Category
+import com.github.HumanLearning2021.HumanLearningApp.datamodel.Picture
+import com.google.firebase.firestore.ktx.firestore
+import com.google.firebase.firestore.ktx.toObject
+import com.google.firebase.ktx.Firebase
+import junit.framework.TestCase
+
+class DatabaseTest : TestCase() {
```

Comment:
Should this really be in app/src/androidTest ? because there is also app/src/test which is maybe more adapted because this test is somewhat independent of android. What do you think?

Reply:
It requires an authenticated connection to the database. Currently the only way to obtain that is through Android.